chia/full_node/full_node.py

```diff
@@ -1714,9 +1714,9 @@ async def _finish_sync(self, fork_point: Optional[int]) -> None:
             peak_fb: Optional[FullBlock] = await self.blockchain.get_full_peak()
             if peak_fb is not None:
                 if fork_point is None:
-                    fork_point = peak_fb.height - 1
+                    fork_point = uint32(max(peak_fb.height - 1, 0))
```

Comment:
I recognize that you didn't change this behavior. But doesn't this look a bit suspicious? if the peak is block 0, should the `fork_point` really also be 0?